content/src/view.js

```diff
@@ -1324,7 +1324,44 @@ $(window).on('resize.listing', function() {
   }
 });
 
-function updatePaneLinks(pane) {
+
+var searchCache={};
+var prevSearch;
+function updateSearchResults(search,pane){
+    search=search.toLowerCase();
+    if(search==prevSearch)
+        return;
+    
+    if(!state.pane[pane].allLinks)
+        state.pane[pane].allLinks=state.pane[pane].links;
+    
+    var results;
+    if(search){
+        if(searchCache[search])
+            results=searchCache[search];
+        else{
```

Comment:
Similar styling comments.  two space indent, space after keyword if, space between `) {`, space around `=`, and always use curly braces.  In this case:

`} else {`